api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -159,25 +162,25 @@ func (c *imageBuilder) getHashedModelDependenciesUrl(ctx context.Context, versio
 		return "", err
 	}
 
-	condaEnvUrl := fmt.Sprintf("gs://%s/%s/model/conda.yaml", artifactURL.Bucket, artifactURL.Object)
+	condaEnvUrl := fmt.Sprintf("%s://%s/%s/model/conda.yaml", c.artifactService.GetURLScheme(), artifactURL.Bucket, artifactURL.Object)
 
 	condaEnv, err := c.artifactService.ReadArtifact(ctx, condaEnvUrl)
 	if err != nil {
 		return "", err
 	}
 
 	hash := sha256.New()
-	hash.Write([]byte(condaEnv))
+	hash.Write(condaEnv)
```

Comment:
The conversion of `condaEnv` to bytes is unnecessary since it's already of the type `[]byte`.